### PR TITLE
Bug fixes.

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

**Comment by gbasood** [comment]
this will break something
just so I can say i told you so

File: code/game/objects/effects/decals/cleanable.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /obj/effect/decal/cleanable
 	var/list/random_icon_states = list()
 	var/targeted_by = null	//Used so cleanbots can claim a mess.
-	mouse_opacity = 1 //N3X made this 0, which made it impossible to click things, and in the current 510 version right-click things.
+	mouse_opacity = 0 //N3X made this 0, which made it impossible to click things, and in the current 510 version right-click things.
```
Comment (Exxion) [inline code/game/objects/effects/decals/cleanable.dm:4]: The comment seems pretty unnecessary now

**Comment by clusterfack** [comment]
@gbasood why did you change it to this anyway

**Comment by gbasood** [comment]
Wanted to mirror old behavior
But it wasnt important so its fine

**Pull request merged**